odoo/addons/base/models/res_partner.py

```diff
@@ -458,6 +459,15 @@ def _compute_company_registry(self):
         for company in self:
             company.company_registry = company.company_registry
 
+    @api.model
+    def _get_company_registry_placeholder(self, country_code):
```

Comment:
You probably don't need this method. You can compute the placeholder in `_compute_company_registry_placeholder` and only overwrite that method..

Reply:
It's needed by `_compute_company_registry_placeholder` from `res.company` too, we cannot use related fields here imo

Reply:
Why would you have a `company` where `company.company_registry_placeholder != company.partner_id.company_registry_placeholder`?
You can just overwrite `_compute_company_registry_placeholder` on res.partner.

Reply:
indeed thanks!